go.mod

```diff
@@ -46,6 +48,7 @@ require (
 	k8s.io/component-base v0.28.9
 	k8s.io/component-helpers v0.28.9
 	k8s.io/controller-manager v0.28.9
+	k8s.io/klog v1.0.0
```

Comment:
please use k8s.io/klog/v2 instead of importing k8s.io/klog

Reply:
done.